modules/mono/glue/GodotSharp/GodotSharp/Core/Color.cs

```diff
@@ -597,7 +643,7 @@ public Color(string code, float alpha)
         /// <exception name="ArgumentOutOfRangeException">
         /// <paramref name="rgba"/> color code is invalid.
         /// </exception>
-        private static Color FromHTML(ReadOnlySpan<char> rgba)
+        public static Color FromHTML(ReadOnlySpan<char> rgba)
```

Comment:
We may want to also implement an overload that takes String for .NET languages that don't support Spans (I think F# supports them, so it's probably only VB).

Reply:
Because, AFAIK, VB doesn't support ref structs so the method can't be called.